code/src/containers/tables/orders-table/components/orders-table-head/OrdersTableHead.tsx

```diff
@@ -1,15 +1,75 @@
+import { useState } from "react";
+import { useSearchParams } from "react-router-dom";
+
+import { initialSortOrder } from "@/containers/tables/orders-table/components/orders-table-head/OrdersTableHead.constants";
+
 import { AppTableCell } from "@/components/app-table/components";
+import AppTableSortLabel from "@/components/app-table/components/app-table-sort-label/AppTableSortLabel";
 import AppTypography from "@/components/app-typography/AppTypography";
 
+import { SortOrder } from "@/types/common";
+
 import "@/containers/tables/orders-table/components/orders-table-head/OrdersTableHead.scss";
 
 type OrderTableHeadProps = {
   head: string;
 };
+
+const getSortKey = (head: string): string | null => {
+  switch (head) {
+    case "ordersTable.columns.createdAt":
+      return "createdAt";
+    case "ordersTable.columns.totalPrice":
+      return "total";
+    case "ordersTable.columns.status":
+      return "orderStatus";
+    case "ordersTable.columns.isPaid":
+      return "isPaid";
+    default:
+      return null;
+  }
+};
+
```

Comment:
i suggest to use object for that and get this values by head as a key

Reply:
Changed